### ISSUE TITLE
FuturePack/FoamFix Crash

### ISSUE THREAD

**Issue by rbills02** [mcenderdragon/Futurepack-API#897]
I keep getting a rendering screen error while working on FuturePack stuff. Seems to be a FoamFix issue. Is there any work around for this without deleting FoamFix?

https://pastebin.com/Eb2iy4wX

**Comment by rbills02** [comment]
Removed FoamFix and still getting rendering screen error.

https://pastebin.com/HqYt1qtd

**Comment by mcenderdragon** [comment]
``tyra314.toolprogression`` is crashing trying to set THrusters metas to 13 which does not exist.

**Comment by rbills02** [comment]
Yeah I finally realized it happen when I hover over a thruster. Anyway to fix it?

**Comment by mcenderdragon** [comment]
change the tooltip making mod ? there is nothing I can do to fix this.
